warehouse/ingest-core/src/main/java/datawave/ingest/data/config/ingest/BaseIngestHelper.java

```diff
@@ -662,6 +662,22 @@ protected NormalizedContentInterface normalize(NormalizedContentInterface normal
         return list;
     }
     
+    public HashSet<NormalizedContentInterface> normalizeFieldValue(String fieldName, NormalizedContentInterface normalizedContent, boolean indexOnly) {
+        Collection<datawave.data.type.Type<?>> dataTypes = getDataTypes(fieldName);
+        HashSet<NormalizedContentInterface> values = new HashSet<>(dataTypes.size());
+        for (datawave.data.type.Type<?> dataType : dataTypes) {
+            NormalizedContentInterface value = normalizeFieldValue(normalizedContent, dataType);
+            if (indexOnly) {
+                value.setEventFieldValue(null);
+            }
+            values.add(value);
+            if (log.isDebugEnabled()) {
+                log.debug("added normalized field " + normalizedContent + " to values " + values);
```

Comment:
This log line is going to grow for every value we add. So it'll start as `added field A to [A]`, then `added field B to [A B]`, then  'added field C to [A B C]`... I'd opt to either log `normalizedContent` by itself on each iteration or to log the whole set after the loop.

Reply:
fixed in the next commit.